doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -806,7 +806,12 @@ Bootloader libraries
 Debug libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`mod_memfault` library:
+
+  * Fixed an issue the library reset the LTE connectivity statistics after each read.
+    This could lead to an accumulated error over time because of the byte counter resolution.
+    The connectivity statistics are now only reset when the library is initialized and will
+    be cumulative after that.
```

Comment:
```suggestion
  * Fixed an issue where the library resets the LTE connectivity statistics after each read.
    This could lead to an accumulated error over time because of the byte counter resolution.
    The connectivity statistics are now only reset when the library is initialized and will be cumulative after that.
```

Reply:
Seems like the changes are not updated.

Reply:
That was weird, I think it's okay now